### PR TITLE
Improved Quiz Flow

### PR DESCRIPTION
**Pull request by thedevelopersanjeev**
Fixes part 1 of #181 

Changes: 
1. Submit Button Removed from already submitted quizzes
2. Quiz Result Screen does not appear in already submitted quizzes
3. Back button working on the quiz result screen.

Screenshots for the change:
already submitted quiz...
Screenshot_1558510006

### CODE REVIEW

**Comment by thedevelopersanjeev** [comment]
I am not sure of the animations, how should I approach that?
Instead of changing the animation for questions pager adapter, we can invert the animations for every fragment, 
after that, every animation will be from right to left.
@aggarwalpulkit596

**Comment by aggarwalpulkit596** [comment]
yeah revert animation

**Comment by aggarwalpulkit596** [comment]
and make a gif or video for complete flow

**Comment by thedevelopersanjeev** [comment]
video of complete flow: https://drive.google.com/open?id=1PtUXlSR9ZUm5oYN51w8Hx_o7JsLQLk3C
@aggarwalpulkit596

File: app/src/main/java/com/codingblocks/cbonlineapp/fragments/QuizResultFragment.kt [hunk 1]
```diff
@@ -26,6 +32,9 @@ class QuizResultFragment : Fragment() {
     override fun onActivityCreated(savedInstanceState: Bundle?) {
         super.onActivityCreated(savedInstanceState)
         arguments?.let {
+            quizId = it.getString(QUIZ_ID)!!
```
Comment (aggarwalpulkit596) [inline app/src/main/java/com/codingblocks/cbonlineapp/fragments/QuizResultFragment.kt]: don't use not null assertion anywhere

Comment (thedevelopersanjeev) [reply app/src/main/java/com/codingblocks/cbonlineapp/fragments/QuizResultFragment.kt]: done.

**Comment by thedevelopersanjeev** [comment]
please review @aggarwalpulkit596

**Comment by aggarwalpulkit596** [comment]
@thedevelopersanjeev 400 would be sufficient I guess as there was not much to do

**Comment by thedevelopersanjeev** [comment]
Thanks, @aggarwalpulkit596 : )

**Pull request merged**